src/c++/perf_analyzer/genai-perf/tests/test_llm_metrics.py

```diff
@@ -274,11 +274,11 @@ def test_openai_llm_profile_data(self, mock_read_write: pytest.MonkeyPatch) -> N
 
         Metrics
         * time to first tokens
-            - experiment 1: [3 - 1, 4 - 2] = [2, 2]
+            - experiment 1: [5 - 1, 7 - 2] = [4, 5]
         * inter token latencies
-            - experiment 1: [[(5 - 3)/1, (8 - 5)/1, (12 - 8)/1], [(7 - 4)/1, (11 - 7)/3, (15 - 11)/2]]
-                          : [[2, 3, 4], [3, 4/3, 2]]
-                          : [[2, 3, 4], [3, 1, 2]]  # rounded
+            - experiment 1: [[(8 - 5)/1, (12 - 8)/1], [(11 - 7)/3, (15 - 11)/2]]
+                          : [[3, 4], [4/3, 2]]
+                          : [[3, 4], [1, 2]]  # rounded
         * output token throughputs per request
             - experiment 1: [3/(12 - 1), 6/(15 - 2)] = [3/11, 6/13]
```

Comment:
@nv-hwoo can you confirm that none of the output token calculations in this test should change, because the request that we are now ignoring had no tokens?